docs_requirements.txt

```diff
@@ -40,6 +40,7 @@ notebook==6.1.5
 numpy==1.19.3
 packaging==20.7
 pandas==1.1.5
+xarray==0.16.2
 pandocfilters==1.4.3
```

Comment:
Please sort .txt files alphabetically. Xarray is already listed under X in this file :)

Reply:
will do!